apps/account-api/src/api.config.ts

```diff
@@ -1,18 +1,10 @@
 import { JoiUtils } from '#config';
-import { EnvironmentType } from '@projectlibertylabs/graph-sdk';
 import { registerAs } from '@nestjs/config';
 import Joi from 'joi';
 
-export interface IAccountApiConfig {
-  apiBodyJsonLimit: string;
-  apiPort: number;
-  apiTimeoutMs: number;
-  siwfNodeRpcUrl: URL;
-  graphEnvironmentType: keyof EnvironmentType;
-  siwfUrl: string;
-  siwfV2Url?: string;
-  siwfV2URIValidation?: string[];
-}
+import { IAccountApiConfig } from '#types/interfaces/account/api-config.interface';
+
+export { IAccountApiConfig };
```

Comment:
Question: what's the purpose of importing and then exporting this interface?

Reply:
A pattern we use pretty commonly in the codebase is importing the interface and config together.

This was a way to minimizing changing all the imports. Since the `healthCheckModule` is using the config interfaces, the interface was extracted out (e.g. #types/interfaces/account/api-config.interface';). Reexporting it here was a way to pass it along to where it has been imported.

https://github.com/ProjectLibertyLabs/gateway/blob/main/apps/account-api/src/services/accounts.service.ts#L20

Open to any suggestions to improv the encapsulation of the module/clean up the imports.